src/main/java/com/bieganski/ox/ConsoleInterface.java

```diff
@@ -0,0 +1,20 @@
+package com.bieganski.ox;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.Scanner;
+
+public class ConsoleInterface implements UserInterface {
+    private Scanner scanner;
+    private PrintStream out;
+
+    public ConsoleInterface(InputStream inputStream, PrintStream printStream) {
+        this.scanner = new Scanner(inputStream);
```

Comment:
You can't replace scanner now with something else if you will need it.
I am looking forward to tests of your console interface, with that said. :-)